aemedge/blocks/channel-lookup/channel-lookup.js

```diff
@@ -0,0 +1,114 @@
+import { createTag } from '../../scripts/utils.js';
+
+const ZIPCODE_ENDPOINT = 'https://p-geo.movetv.com/geo';
+const DEFAULT_ZIPCODE = '90020';
+const ZIPCODE_KEY = 'user_zip';
+let gqlResponse = '';
+
+export async function getZipcode() {
+  let zipcode = localStorage.getItem(ZIPCODE_KEY);
+  if (!zipcode) {
+    const response = await fetch(ZIPCODE_ENDPOINT);
+    const data = await response.json();
+    zipcode = data?.zip_code || DEFAULT_ZIPCODE;
+    localStorage.setItem(ZIPCODE_KEY, zipcode);
+  }
+  return zipcode;
+}
+function showWarningMessage() {
+  return `
+    <div type="warning" class="warning channel-data">
+      <img src="/aemedge/icons/warning-icon.svg" id="icon" class="warning-icon"/>
+      <h3 class="warning-heading">
+      <span>
+      <p class="warning-msg">Local programming
```

Comment:
could you please add the functionality to load this error message from placeholders spread sheet so that authors can be able to make the change to error message. docs - https://www.aem.live/developer/placeholders .

Reply:
done, please review